src/Widgets/DataFormBooleanWidget/DataFormBooleanWidgetClass.ts

```diff
@@ -5,10 +5,14 @@ export const DataFormBooleanWidget = provideWidgetClass(
   {
     attributes: {
       attributeName: 'string',
+      dataForNo: 'string',
+      dataForYes: 'string',
```

Comment:
Why do we need something complicated like `dataForNo` or `dataForYes`?

Does Neoletter really have such a complex API? Isn't it a simple boolean?

Reply:
Generally, if something is called ` BooleanWidgetClass`, shouldn't it use booleans?

If the neoletter API needs "special words" instead of booleans, that would be the responsiblity of the data connection.

Reply:
I think this was an intermediate step. Please see https://github.com/Scrivito/scrivito-portal-app/pull/289#discussion_r1492238961 .